src/istio/control/attribute_names.cc

```diff
@@ -28,15 +28,17 @@ const char AttributeName::kRequestMethod[] = "request.method";
 const char AttributeName::kRequestPath[] = "request.path";
 const char AttributeName::kRequestReferer[] = "request.referer";
 const char AttributeName::kRequestScheme[] = "request.scheme";
-const char AttributeName::kRequestSize[] = "request.size";
+const char AttributeName::kRequestBodySize[] = "request.size";
+const char AttributeName::kRequestTotleSize[] = "request.totle_size";
 const char AttributeName::kRequestTime[] = "request.time";
 const char AttributeName::kRequestUserAgent[] = "request.useragent";
 const char AttributeName::kRequestApiKey[] = "request.api_key";
 
 const char AttributeName::kResponseCode[] = "response.code";
 const char AttributeName::kResponseDuration[] = "response.duration";
 const char AttributeName::kResponseHeaders[] = "response.headers";
-const char AttributeName::kResponseSize[] = "response.size";
+const char AttributeName::kResponseBodySize[] = "response.size";
+const char AttributeName::kResponseTotleSize[] = "response.totle_size";
```

Comment:
sorry to be late with this comment, but should these attributes be `request.total_size` and `response.total_size` ?

Reply:
request.total_size is added at line 32. Prior to this PR, we already let Mixer filter to report request.size and response.size, and they are actually request body size and response body size. So I update the variable name (e.g. s/kResponseSize/kResponseBodySize) to make it less confusing.

Reply:
@JimmyCYJ I'm actually wondering about the difference between `totle` and `total` (and what the name of the attribute is).

Reply:
That is typo. I will fix it soon. Thanks!